src/partials/contacts.html

```diff
@@ -0,0 +1,72 @@
+      <section class="contact">
+        <h2>Contacts</h2>
+        <!--VISUALLY HIDDEN-->
+        <ul class="contacts-list">
+          <li class="contacts-main-item">
+            <p class="contacts-place-type">CAFE</p>
+            <p class="contacts-city">
+              Chicago
+              <a href="#" class="contacts-adress"
+                >10340 Ridgeland Ave, Chicago Ridge, IL 60415, USA</a
+              >
+              <!--link do gmaps w #-->
+            </p>
+            <p class="contacts-workhours">
+              Monday - Friday <time class="contacts-workhours-wrapper" data-time="06:00 AM - 10:00 PM">06:00 AM - 10:00 PM</time>
+            </p>
+            <p class="contacts-workhours">
+              Saturday - Sunday <time class="contacts-workhours-wrapper" data-time="08:00 AM - 16:00 PM">08:00 AM - 16:00 PM</time>
```

Comment:
to atrybut datetime nie data-time. I musi to wyglądać tak:
<p class="contacts-workhours">
  Saturday - Sunday
  <time class="contacts-workhours-wrapper" datetime="08:00">08:00 AM</time> -
  <time class="contacts-workhours-wrapper" datetime="08:00">16:00 PM</time> 
</p>